### ISSUE TITLE
Deep property fill not working

### ISSUE THREAD

**Issue by daviatorstorm** [MisterJames/GenFu#73]
SomeClass.cs
```cs
class IdEntity<T> {
public T Id { get; set; }
}

class NamedEntity : IdEntity<int> {
public string Name { get; set; }
}

public class Entity: NamedEntity {
}
```

Test.cs
```cs
using GenFu;
using XUnit;

class EntityFill {
[Fact]
public void EntityFilling() {
A.Configure<Entity>()
.Fill(x => x.Id, () => 0)
.Fill(x => x.Name, () => "Value");

var testData = A.ListOf<Entity>(3);
// Test data not using above configure generating own random values
}
}
```

**Comment by dpaquette** [comment]
This appears to be related to #25

**Comment by MisterJames** [comment]
@daviatorstorm I believe this is fixed with last week's release, which incorporates the fix in #25. Please let us know if you're still running into it.
